src/communication/azext_communication/manual/_params.py

```diff
@@ -255,10 +255,10 @@ def _load_email_arguments(self):
         c.argument('subject', options_list=['--subject'], type=str, help='Subject of the email message.')
         c.argument('text', options_list=['--text'], type=str, help='Plain text version of the email message. Optional.')
         c.argument('html', options_list=['--html'], type=str, help='Html version of the email message. Optional.')
-        c.argument('importance', options_list=['--importance'], arg_type=get_enum_type(['normal', 'low', 'high']),
-                   help='The importance type for the email. Known values are: high,'
-                   ' normal, and low. Default is normal. Optional')
         c.argument('recipients_to', options_list=['--to'], nargs='+', help='Recepients email addresses.')
+        c.argument('importance', options_list=['--importance'], arg_type=get_enum_type(['1','2','3','4','5']),
+                   help='The importance type for the email. Possible values include: 1(High), 2(High),'
+                   '3(Normal), 4(Low), 5(Low). Optional.')
```

Comment:
We should only provide 3 possible values to customer - high, normal and low (as it used to be before). In the implementation, you should convert it to a number from 1 to 5 and send it in the custom header.

Reply:
Makes sense. Updating.